templates/library.html

```diff
@@ -45,12 +45,17 @@ <h1 style="margin-top: 00px;">{{library.name}}</h1>
           <table style="margin-top: 10px;" cellpadding="0" cellspacing="0" border="0" class="table table-striped" id="example">
             <tbody>
               {{#files}}
-              <tr class="library {{fileType}}-type">
+              <tr class="library {{fileType}}-type {{#isHidden}}hidden{{/isHidden}}">
                 <td class="library-column"  data-lib-name="{{library.name}}" style="position: relative;">
                   <p class='library-url'>https://cdnjs.cloudflare.com/ajax/libs/{{library.originalName}}/{{version}}/{{name}}</p>
                 </td>
               </tr>
               {{/files}}
+              <tr>
+                <td colspan="2">
+                  <a class="show-hidden-files" href="#">Show all files</a>
```

Comment:
hmmm ... I doubt if there is performance improvement, use material-design-icons as an example, the large file list still takes a long time to be loaded, though it's invisible, but it is true to improve the usability. This may be another issue, but maybe we can add a animate pic to tell the users the data is under loading.